### PR TITLE
Parallelize

### LINKED ISSUE #5: Make program concurent

### PR DESCRIPTION
**Pull request by codefloww**
Closes #5

### CODE REVIEW

**Review by andylvua — APPROVED** [review]

File: main.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #include "app/app.hpp"
 
-int main() {
+int main(int argc, char* argv[]) {
```
Comment (codefloww) [inline main.cpp]: Don't understate the purpose of the cl arguments. Consider removing or adding some useful functionality

File: src/app/scene.cpp [hunk 2]
```diff
@@ -54,41 +56,47 @@ Scene::Scene() {
...
+                    glm::vec3 loc_normal;
+                    glm::vec3 loc_color;
+                    double x_factor = 1.0 / (static_cast<double>(width) / 2.0);
+                    double y_factor = 1.0 / (static_cast<double>(height) / 2.0);
+                    double norm_x = static_cast<double>(x) * x_factor - 1.0;
+                    double norm_y = static_cast<double>(y) * y_factor - 1.0;
+                    camera_m.createRay(norm_x, norm_y, camera_ray);
+                    internalRender(x, y, camera_ray, output_image, int_point, loc_normal, loc_color);
+                }
+            }
```
Comment (codefloww) [inline src/app/scene.cpp]: Noice

**Review by codefloww — COMMENTED** [review]
Overall looks okay

File: src/app/scene.cpp (hunk 2 above)
Comment (andylvua) [reply src/app/scene.cpp]: Yup

File: main.cpp (hunk 1 above)
Comment (andylvua) [reply main.cpp]: @SmalRat Please review your change

**Review by andylvua — APPROVED** [review]

**Pull request merged**